### PR TITLE
Us80 86

### PR DESCRIPTION
**Pull request by koulyves**
Add Menu option with Login and registration UI

### CODE REVIEW

File: simoc_server/templates/panel_content.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <div class="pt-5">
+        <p><button type="button" id="loginbtn" class="btn btn-primary">Login</button></p>
+        <p><button type="button" id="registerbtn" class="btn btn-primary ">Register</button></p>
+        <p><button type="button" class="btn btn-primary ">Exit</button></p>
+
+    </div>
+
+</div>
+
+<script src="{{ url_for('static', filename='js/jquery/jquery.min.js') }}"></script>
```
Comment (bdmccord) [inline simoc_server/templates/panel_content.html]: This is imported twice, once in base_home and once here

File: simoc_server/static/css/custom/main.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  font-family:'jupiter';
+  color:#ffffff;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  background-size: cover;
+  -o-background-size: cover;
+}
+@font-face{
+    font-family:'jupiter';
+    src:url('../../assets/fonts/jupiter.ttf')
```
Comment (bdmccord) [inline simoc_server/static/css/custom/main.css]: need to fix case of imports (this one should be Jupiter.ttf)

File: simoc_server/static/css/custom/main.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+}
+div .row .sf_window{
+   width:500px;
+   height:500px;
+   margin:0 auto;
+   background:url('../../assets/img/bg/sf_window.png') no-repeat;
```
Comment (bdmccord) [inline simoc_server/static/css/custom/main.css]: This one should be SF_window.png. There may be others

File: simoc_server/views.py [hunk 4]
```diff
@@ -4,10 +4,10 @@
 from uuid import uuid4
 
 from flask_login import LoginManager, login_user, login_required, logout_user, current_user
-from flask import request, session, send_from_directory, safe_join
+from flask import request, session, send_from_directory, safe_join, Flask, render_template
```
[the remark this answers is not in the document]
Comment (koulyves) [reply simoc_server/views.py]: Don't really need Flask here. just needed render_template. removing.

**Pull request merged**